frontend/public/components/overview/pods-overview.tsx

```diff
@@ -140,44 +140,47 @@ export const PodsOverviewContent: React.SFC<PodsOverviewContentProps> = ({
   const {
     metadata: { name, namespace },
   } = obj;
-
+  const { t } = useTranslation();
   const [showWaitingPods, setShowWaitingPods] = React.useState(false);
   const showWaitingForBuildAlert =
-    hasBuildConfig && isDeploymentGeneratedByWebConsole(obj) && pods.some(isPodWithoutImageId);
+    hasBuildConfig && isDeploymentGeneratedByWebConsole(obj) && !pods.some(isPodWithImageId);
 
   let filteredPods = [...pods];
   if (showWaitingForBuildAlert && !showWaitingPods) {
-    filteredPods = filteredPods.filter((pod) => !isPodWithoutImageId(pod));
+    filteredPods = filteredPods.filter((pod) => isPodWithImageId(pod));
   }
   filteredPods.sort(podCompare);
 
   const errorPodCount = _.size(_.filter(pods, (pod) => isPodError(pod)));
   const podsShown = Math.max(Math.min(errorPodCount, MAX_ERROR_PODS), MAX_PODS);
   const linkTo = allPodsLink || `${resourcePath(referenceFor(obj), name, namespace)}/pods`;
-  const emptyMessage = emptyText || 'No Pods found for this resource.';
+  const emptyMessage = emptyText || t('public~No Pods found for this resource.');
 
   return (
     <>
-      <SidebarSectionHeading text="Pods">
+      <SidebarSectionHeading text={t('public~Pods')}>
         {_.size(pods) > podsShown && (
           <Link className="sidebar__section-view-all" to={linkTo}>
-            {`View all (${_.size(pods)})`}
+            {t('public~View all {{podSize}}', { podSize: _.size(pods) })}
           </Link>
         )}
       </SidebarSectionHeading>
       {showWaitingForBuildAlert ? (
         <Alert
           isInline
           variant="info"
-          title="Waiting for the build"
+          title={t('public~Waiting for the build')}
           actionLinks={
             <AlertActionLink onClick={() => setShowWaitingPods(!showWaitingPods)}>
-              {`${showWaitingPods ? 'Hide' : 'Show'} waiting pods with errors`}
+              {t('public~{{waitingPods}} waiting pods with errors', {
+                waitingPods: showWaitingPods ? 'Hide' : 'Show',
+              })}
```

Comment:
Hi, this doesn't translate Hide and Show, right? So my suggest is:

```suggestion
              {showWaitingPods ? t('public~Hide waiting pods with errors') : t('public~Show waiting pods with errors')}
```

Reply:
I notice that your translation works fine, but its maybe still easier to translate two separate messages.

Reply:
Done